website/markdown/posts/2020-01-31-version-1.2.0/post.mdx

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Tuist 1.2.0 Bugfixes and improvements
+categories: [tuist, release, swift, 1.2.0, project generation, xcode, swift]
+excerpt: Maintenance release containing many bugfixes and improvements. A new section about Tuist's best practices is also a welcome addition to the website.
+author: vytis
+---
+
+We are slowly learning how to do smaller [bi-weekly releases](https://tuist.io/docs/contribution/release/#cadence). Everything went (almost) according to plan and Tuist 1.2.0 is here for you to enjoy!
+
+## New contributors
+
+Firstly, warm welcome to another first-time Tuist contributor [Maciej Piotrowski](https://github.com/paciej00) 👏👏👏. He improved the API introduced in the previous release for excluding files from a target. It makes exclude patterns more flexible which can be very useful when migrating an existing project to Tuist!
+
+## TBD
+
+TBD, maybe a few sentences about fixture generator?
```

Comment:
I think it's a good idea. We can mention that we disregarded performance for a long time and now we'd like to tackle it. For that we developed a fixture generator to generate projects that we can use from performance tests.